catalog-data.ttl

```diff
@@ -4151,3 +4154,12 @@ cdata:Zhilin_Zhang a ex:Person ;
 cdata:Zongyan_Li a ex:Person ;
   ex:name "Zongyan Li" ;
   ex:subType con:Person .
+
+<urn:uuid:3b05c17c-f44a-470e-8031-408abee6e2ca> a ex:Person ;
+  ex:name "Eric Prud'hommeaux" .
+
+<urn:uuid:43a1b70e-f40c-49d6-baad-f070ed9ff9e2> a ex:Person ;
+  ex:name "damooo" .
+
+<urn:uuid:d1e0e382-0e30-483e-b4b4-13d12299994e> a ex:Person ;
```

Comment:
@potherca what is your WebId? Are there other handles you'd like to add whilst you're here?